app/src/main/kotlin/com/wire/android/mapper/UICallParticipantMapper.kt

```diff
@@ -39,6 +39,7 @@ class UICallParticipantMapper @Inject constructor(
         isCameraOn = participant.isCameraOn,
         isSharingScreen = participant.isSharingScreen,
         avatar = participant.avatarAssetId?.let { ImageAsset.UserAvatarAsset(wireSessionImageLoader, it) },
-        membership = userTypeMapper.toMembership(participant.userType)
+        membership = userTypeMapper.toMembership(participant.userType),
+        hasEstablishedAudio = participant.hasEstablishedAudio
```

Comment:
Can we write some test for it?

Reply:
it's already tested in `UICallParticipantMapperTest.kt`